src/pages/workspace/companyCards/WorkspaceCompanyCardDetailsPage.tsx

```diff
@@ -169,6 +170,18 @@ function WorkspaceCompanyCardDetailsPage({route}: WorkspaceCompanyCardDetailsPag
                                 title={card?.scrapeMinDate ? format(parseISO(card.scrapeMinDate), CONST.DATE.FNS_FORMAT_STRING) : ''}
                                 interactive={false}
                             />
+                            <MenuItem
+                                icon={Expensicons.MoneySearch}
+                                title={translate('workspace.common.viewTransactions')}
+                                style={styles.mv1}
```

Comment:
All of these menu items should not have a gap in between them, so I think we need to remove the vertical margin here. Can you make sure all of the menu items are 64px tall with no gaps between? Screenshots would be great, thanks! cc @Expensify/design for viz

Reply:
Doesn't quite look right. We want a gap of 12px below the last push row and above the block of optionRows. They should appear like two distinct sections. It's just that we want all of the option rows in the same block to not have any vertical margin. 

![CleanShot 2025-01-15 at 09 52 14@2x](https://github.com/user-attachments/assets/4f329f32-0ead-4a45-8e93-d469bf77646b)


For the option rows on your second screenshot, why do some have a green icon and others a gray icon? I think we want them all to be gray.

Reply:
Default state would be gray, but then they are green on hover:
![CleanShot 2025-01-15 at 10 33 20](https://github.com/user-attachments/assets/e79a8639-80df-4197-abd4-bb930d14b9e4)

Reply:
Lovely, thanks!